scripts/fmt/format-code.sh

```
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+# format-code.sh — Auto-format C++ source files using clang-format.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+if [[ "${TRACE-0}" == "1" ]]; then set -o xtrace; fi
+
+echo "==> formatting C++ code..."
+find src -name '*.cpp' -o -name '*.h' | xargs clang-format -i --style=file:.config/.clang-format
```

Shellcheck (0.11.0)

[warning] Use 'find .. -print0 | xargs -0 ..' or 'find .. -exec .. +' to allow non-alphanumeric filenames.

(SC2038)